flocker/common/script.py

```diff
@@ -139,9 +158,10 @@ def main(self):
         self._react(self.script.main, (options,), _reactor=self._reactor)
 
         # Not strictly necessary, but nice cleanup for tests:
-        if observer is not None:
-            removeObserver(observer)
-            log_file.close()
+        # XXX FLOC-936
+        # 0. If self.logging is False, don't do anything. Otherwise:
+        # 1. Remove EliotObserver
+        # 2. Stop ThreadedLogWriter
```

Comment:
:question: If you make some context managers I guess you could use a `with` block here and have slightly neater cleanup.